internal/describer/customresource.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/vmware/octant/pkg/store"
 )
 
+// TODO: delete me
```

Comment:
Is there an issue for this?

Reply:
ha! there should be. let me go do that now.